### PR TITLE
Add documentation for proposed Starlink integration

### PR DESCRIPTION
**Pull request by pacorain**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/62764
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/starlink.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Starlink
+description: Monitor Starlink satellite status from home Assistant
+  - Binary Sensor
+  - Sensor
+ha_release:
```
Comment (frenck) [inline source/_integrations/starlink.markdown:6]: Please add the next possible version.

File: source/_integrations/starlink.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: Starlink
+description: Monitor Starlink satellite status from home Assistant
+  - Binary Sensor
```
Comment (frenck) [inline source/_integrations/starlink.markdown:4]: This is an invalid YAML. I think you meant to add `ha_category`

File: source/_integrations/starlink.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@pacorain'
+ha_domain: starlink
+ha_platforms:
+  - binary_sensor
+  - sensor
+---
+The `starlink` platform consumes data from your [Starlink sattelite](https://www.starlink.com/) into Home Assistant sensors.
```
Comment (frenck) [inline source/_integrations/starlink.markdown:16]: This isn't a platform

File: source/_integrations/starlink.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+{% include integrations/config_flow.md %}
+
+{% configuration %}
```
Comment (frenck) [inline source/_integrations/starlink.markdown:20]: Use the basic configuration instead, `configuration` is meant for YAML-based integrations.

File: source/_integrations/starlink.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  required: true
+  type: string
+  default: Starlink
+{% endconfiguration %}
+
+## Configuring your network
+
+If you are using the Starlink provided router with your dish, the default configuration settings should work just fine without any further configuration.
+
+If you are using an alternate router, you will need to set up a static route. This can usually be done within your router's configuration page.
```
Comment (frenck) [inline source/_integrations/starlink.markdown:42]: This seems odd?

Additionally, I don't think the average user won't even remotely know what this even means.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/starlink.markdown (hunk 5 above)
Comment (LordNex) [reply source/_integrations/starlink.markdown:42]: Unfortunately we do. The router they give us doesn't even have an Ethernet port. You have to buy an add on and shut off/bypass mode the Starlink router. There are. Still stats available if your setup with Ethernet but your behind a CGNAT, so you end up having to route to thier network to get the info from dishy.starlink.com. I'm pretty sure there are some APIs that might be able to be pulled as well. But I'm no programmer

**Comment by frenck** [comment]
Closing the PR as the parent PR has been closed.

**Pull request closed without merging**